From cab8b18f9ac3c1f980e253c71f68130875f59c8e Mon Sep 17 00:00:00 2001
From: Nicola Guerrera <guerrera.nicola@gmail.com>
Date: Sat, 9 Nov 2024 00:22:37 +0100
Subject: [PATCH] tqftpserv.service.in: remove dependency on qrtr-ns.service

QRTR is built in to the kernel now, the other packages (such as rmtfs) have since removed qrtr-ns as a service dependency

Signed-off-by: Nicola Guerrera <guerrera.nicola@gmail.com>
---
 tqftpserv.service.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tqftpserv.service.in b/tqftpserv.service.in
index 3333e02..71a7991 100644
--- a/tqftpserv.service.in
+++ b/tqftpserv.service.in
@@ -1,7 +1,5 @@
 [Unit]
 Description=QRTR TFTP service
-Requires=qrtr-ns.service
-After=qrtr-ns.service
 
 [Service]
 ExecStart=@prefix@/bin/tqftpserv
